src/lib/Modal.svelte

```diff
@@ -1,30 +1,37 @@
 <script lang="ts">
-  // It starts open
-  // on:close is forwarded
-  // Relies on something else for styling (picocss)
-  let dialog: HTMLDialogElement;
+  export let show = false;
```

Comment:
This API is much nicer, thank you. I've been meaning to switch it in all my projects anyway. Pinning to a GH version + TS means this is both not urgent and not really that hard